datasets/atomic/atomic.py

```diff
@@ -0,0 +1,151 @@
+# coding=utf-8
+# Copyright 2020 The HuggingFace Datasets Authors and the current dataset script contributor.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""The Atommic Dataset"""
+
+from __future__ import absolute_import, division, print_function
+
+import json
+import os
+
+import datasets
+
+
+_CITATION = """@article{Sap2019ATOMICAA,
+  title={ATOMIC: An Atlas of Machine Commonsense for If-Then Reasoning},
+  author={Maarten Sap and Ronan Le Bras and Emily Allaway and Chandra Bhagavatula and Nicholas Lourie and Hannah Rashkin and Brendan Roof and Noah A. Smith and Yejin Choi},
+  journal={ArXiv},
+  year={2019},
+  volume={abs/1811.00146}
+}
+"""
+
+
+_DESCRIPTION = """This dataset provides the template sentences and
+relationships defined in the ATOMIC common sense dataset. There are
+three splits - train, test, and dev.
+
+From the authors.
+
+Disclaimer/Content warning: the events in atomic have been
+automatically extracted from blogs, stories and books written at
+various times. The events might depict violent or problematic actions,
+which we left in the corpus for the sake of learning the (probably
+negative but still important) commonsense implications associated with
+the events. We removed a small set of truly out-dated events, but
+might have missed some so please email us (msap@cs.washington.edu) if
+you have any concerns.
+
+"""
+
+_HOMEPAGE = "https://homes.cs.washington.edu/~msap/atomic/"
+
+_LICENSE = "The Creative Commons Attribution 4.0 International License. https://creativecommons.org/licenses/by/4.0/"
+
+_URLs = {"atomic": "https://homes.cs.washington.edu/~msap/atomic/data/atomic_data.tgz"}
+
+
+class Atomic(datasets.GeneratorBasedBuilder):
+    """Atomic Common Sense Dataset"""
+
+    VERSION = datasets.Version("1.1.0")
+
+    BUILDER_CONFIGS = [
+        datasets.BuilderConfig(name="atomic", version=VERSION, description="The Atomic dataset"),
+    ]
+
+    DEFAULT_CONFIG_NAME = "atomic"
```

Comment:
```suggestion
```
not necessary since there's only one config

Reply:
I'll make the change